File: pudgeland/src/main/java/me/elaresai/pudgeland/Pudgeland.java
package me.elaresai.pudgeland;

import org.bukkit.plugin.java.JavaPlugin;

public final class Pudgeland extends JavaPlugin {

    @Override
    public void onEnable() {
        // Plugin startup logic
        getLogger().info("Hello, World!");
    }

    @Override
    public void onDisable() {
        // Plugin shutdown logic
    }
}

File: pudgeland/src/main/java/me/elaresai/pudgeland/Pudgeland.kt
package me.elaresai.pudgeland

import me.elaresai.pudgeland.command.executor.ConnectCommandExecutor
import me.elaresai.pudgeland.command.executor.ConvertCommandExecutor
import org.bukkit.plugin.java.JavaPlugin

class Pudgeland : JavaPlugin() {
    override fun onEnable() {
        // Plugin startup logic
        logger.info("Hello, World!")

        getCommand("connect")?.setExecutor(ConnectCommandExecutor())
        getCommand("convert")?.setExecutor(ConvertCommandExecutor())
    }

    override fun onDisable() {
        // Plugin shutdown logic
    }
}
